tasks/kernels.py

```diff
@@ -0,0 +1,131 @@
+from invoke import task
+from os import makedirs
+from os.path import join, exists
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+from tasks.util.env import PLOTS_FORMAT, PLOTS_ROOT, PROJ_ROOT
+
+RESULTS_DIR = join(PROJ_ROOT, "results", "kernels")
+PLOTS_DIR = join(PLOTS_ROOT, "kernels")
+NATIVE_CSV = join(RESULTS_DIR, "kernels_native.csv")
+WASM_CSV = join(RESULTS_DIR, "kernels_wasm.csv")
+OUT_FILE = join(PLOTS_DIR, "slowdown.png")
+
+
+def _read_results(csv):
+    if not exists(csv):
+        raise RuntimeError("CSV not found: {}".format(csv))
+
+    results = pd.read_csv(csv)
+
+    # First filter only the timing stats, and then group by kernel
+    results = results.loc[results["StatName"] == "Avg time (s)"]
+    results = results.groupby("Kernel", as_index=False)
+
+    # Second, group by world size
+    result_dict = {}
+    for name, group in results:
+        # TODO - how do we present errors in a slowdown plot?
```

Comment:
You can have a look at the `experiment-microbench` plots for some ideas. I'd put an error bar on the bars in the bar chart.

https://github.com/faasm/experiment-microbench/blob/master/tasks/plot.py